components/wb-bubble/wb-bubble-va-en.html

```diff
@@ -0,0 +1,59 @@
+---
+title: Bubble - Open Overlay
+language: en
+category: Plugins
+description: Working example of the virtual assistant bubble with customized avatar.
+altLangPage: wb-bubble-fr
+dateModified: 2021-06-21
+---
+<span class="wb-prettify all-pre hide"></span>
+
+<h2>Virtual assistant bubble including custumized avatar</h2>
+
+<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec aliquam sagittis ante. Nam ac varius tellus, vel laoreet orci. Nullam eget luctus erat. Vivamus mollis ut mauris et iaculis. Donec euismod, nulla et condimentum lobortis, arcu nunc feugiat purus, ut porta quam nulla auctor eros. In iaculis ante id metus semper gravida. Nunc vestibulum, nisi cursus semper feugiat, lorem neque blandit nulla, nec molestie tortor quam a magna. Pellentesque varius arcu id justo congue dictum. Nullam vehicula est nunc, nec tincidunt enim mattis et.</P>
+
+<p>Vestibulum eget quam ut leo laoreet pharetra. Duis a ultrices nisi. Fusce convallis suscipit dolor at tempor. Proin nunc leo, elementum a vulputate eu, sollicitudin at arcu. Fusce id lectus turpis. In ac volutpat tellus. Proin faucibus tellus mollis nisi tempor feugiat. Phasellus quis mi placerat libero facilisis dapibus. Suspendisse et turpis tristique, dignissim mi a, iaculis elit.</P>
+
+<p>Pellentesque ullamcorper quis sapien vel malesuada. Proin laoreet vehicula efficitur. Donec ac cursus elit, a dapibus quam. Suspendisse at urna et ligula vehicula imperdiet non vel nunc. In nibh libero, pellentesque vitae accumsan ac, ornare nec leo. Fusce eros ante, congue in nunc nec, hendrerit pharetra massa. Sed sagittis at tellus vitae semper. Praesent et diam quis ante facilisis consequat vel non mauris. Morbi nibh lorem, ultricies a ullamcorper vitae, condimentum ac metus. Mauris at augue risus. Proin bibendum et mauris ut viverra. Donec rutrum, est sit amet imperdiet rutrum, quam odio ultricies elit, ut vulputate justo velit ac lacus. Nullam interdum rhoncus sem et malesuada. Nullam hendrerit elit malesuada, ultricies nunc a, laoreet nisi. Vivamus vulputate tortor nibh, a pharetra mauris dignissim ut. Fusce sed consectetur orci. </P>
+
+<p>Sed iaculis ex non blandit aliquam. Vivamus tincidunt enim in orci elementum, vel varius est venenatis. Vivamus dapibus vitae nunc eget convallis. Sed vitae est risus. Quisque tincidunt metus a ante eleifend faucibus. Etiam condimentum maximus orci, sit amet vehicula elit elementum eget. Donec vitae consequat mauris. Donec scelerisque mi eu dolor fringilla, quis tempor orci tempor. Ut massa tellus, ultrices vel mattis in, malesuada non arcu. Quisque quis tellus est. Morbi metus enim, pulvinar vitae tempor dictum, sollicitudin ac orci. Nulla sagittis ultricies congue. Curabitur egestas malesuada erat a mattis. </P>
+
+<button type="button" data-wb-doaction="{ 'action': 'openDialog', 'source': '#mid-screen' }" class="wb-bubble" data-wb-bubble="{ 'avatar': 'assets/headset-avatar.png' }">Virtual assistant</button>
```

Comment:
Single and double quotes should be put the other way around, otherwise it is invalid, like so:
```suggestion
<button type="button" data-wb-doaction='{ "action": "openDialog", "source": "#mid-screen" }' class="wb-bubble" data-wb-bubble='{ "avatar": "assets/headset-avatar.png" }'>Virtual assistant</button>
```